packages/lesswrong/components/review/ReviewVotingPage.tsx

```diff
@@ -495,20 +492,14 @@ const ReviewVotingPage = ({classes}: {
                   freeEntry={true}
                 />
               </div>
-              <ReviewPostButton post={expandedPost} year={YEAR+""} reviewMessage={<div>
+              <ReviewPostButton post={expandedPost} year={REVIEW_YEAR+""} reviewMessage={<div>
                 <div className={classes.writeAReview}>
                   <div className={classes.reviewPrompt}>Write a review for "{expandedPost.title}"</div>
                   <div className={classes.fakeTextfield}>Any thoughts about this post you want to share with other voters?</div>
                 </div>
               </div>}/>
 
               <div className={classes.comments}>
-                <PostReviewsAndNominations
-                  title="nomination"
-                  singleLine
-                  terms={{view: NOMINATIONS_VIEW, postId: expandedPost._id}}
-                  post={expandedPost}
-                />
```

Comment:
Should we include the posts you upvoted during that year here, at least during the nominations phase?